src/common/LoggedInProviders.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { ActiveAccountContextProvider } from '../hooks/useActiveAccount';
+
+export const LoggedInProviders = ({
```

Comment:
I'm not sure about this name but it seems fine for now. To me this feels like the one provider that any consumer will have to import to be able to use our services. Something like `LifeomicPlatformProvider` might be appropriate but I am good to kick this down the road.

Reply:
I know what you're saying, but somehow I'd love to convey that anyone is welcome to (and encouraged) re-implement these provider lists.  I could almost see RootProviders & LoggedInProviders being prefixed with `Default` or something.  `LifeomcPlatformProvider` might accidentally convey that it _must_ be used.  Idk- maybe /** */ docs above these are going to be what take this from an blunt import name to a well-explained thing.

Reply:
Also worth mentioning, I think the reason we previously named this as such is because of how it's used:
```
if (isLoggedIn) {
    return (
      <LoggedInProviders>
         ...
```

Whereas if the user is not logged in, these are altogether not in the render/stack.